sirepo_bluesky/sirepo_detector.py

```diff
@@ -116,6 +120,16 @@ def trigger(self):
                                                    'title',
                                                    self.parents['par' + str(i)])
                     element[real_field] = x
+                    if self.parents[f'par{i}'] in self.autocompute_params.keys() and 'grazingAngle' in dict_key:
+                        grazing_vecs_dict = {}
+                        autocompute_key = f'{self.parents[f"par{i}"]}_sirepo_autocomputeVectors'
```

Comment:
Wow, f-string within another f-string, never saw it before, but I hope it works as expected!